### DISCUSSION TITLE
Tests are beginning!

### DISCUSSION THREAD

**Discussion by Marc92020** [ScratchNitro/Scratch-Nitro#2] [Annoucements]
We will now be testing code!

**Comment by JackyBorderCollie** [comment]
Cool! I made also a popup using JavaScript. I found a website that converst Scratch projects to JavaScript, and then I have downloaded it :)

**Comment by JackyBorderCollie** [comment]
The yellow one is JavaScript :)
As you can see, in the repository, there is a bit of JavaScript
![](https://u.cubeupload.com/JackyBorderCollie/languagesRepository.png)

**Comment by JackyBorderCollie** [comment]
Moved to Annoucements!
